<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.qmetry.qaf.automation.step.client.DataDrivenScenario" tests="1" time="18.449" ignored="1" failures="0" timestamp="23 Oct 2019 11:02:50 GMT" skipped="0" hostname="ICPL13076" errors="1">
  <testcase name="user is able to enter all the data on find a flight page successfully" time="18.449" classname="com.qmetry.qaf.automation.step.client.DataDrivenScenario">
    <error type="org.openqa.selenium.TimeoutException" message="Timed out after 3 seconds: Wait time out for By.xpath: //input[@name=&#039;userName&#039;] to be present
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;ICPL13076&#039;, ip: &#039;10.11.24.4&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_201&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 3 seconds: Wait time out for By.xpath: //input[@name='userName'] to be present
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'ICPL13076', ip: '10.11.24.4', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_201'
Driver info: driver.version: unknown
	at .When  user login with valid '${signin.data.username}' and valid '${signin.data.password}'(resources/scenarios/FindAFlight.bdd:4)
	at com.qmetry.qaf.automation.ui.util.DynamicWait.until(DynamicWait.java:170)
	at com.qmetry.qaf.automation.ui.webdriver.QAFExtendedWebElement.waitForPresent(QAFExtendedWebElement.java:559)
	at com.qmetry.qaf.automation.step.CommonStep.waitForPresent_aroundBody50(CommonStep.java:643)
	at com.qmetry.qaf.automation.step.JavaStepReporter$MockJavaStep.doExecute(JavaStepReporter.java:138)
	at com.qmetry.qaf.automation.step.BaseTestStep.execute(BaseTestStep.java:149)
	at com.qmetry.qaf.automation.step.JavaStepReporter.javaTestStep(JavaStepReporter.java:88)
	at com.qmetry.qaf.automation.step.CommonStep.waitForPresent(CommonStep.java:642)
	at com.newtour.steps.LoginBDDSteps.doLogin_aroundBody2(LoginBDDSteps.java:15)
	at com.qmetry.qaf.automation.step.JavaStepReporter.javaTestStep(JavaStepReporter.java:101)
	at com.newtour.steps.LoginBDDSteps.doLogin(LoginBDDSteps.java:13)
	at com.qmetry.qaf.automation.step.JavaStep.doExecute(JavaStep.java:153)
	at com.qmetry.qaf.automation.step.BaseTestStep.execute(BaseTestStep.java:149)
	at com.qmetry.qaf.automation.step.StringTestStep.execute(StringTestStep.java:134)
	at com.qmetry.qaf.automation.step.client.Scenario.execute(Scenario.java:180)
	at com.qmetry.qaf.automation.step.client.DataDrivenScenario.scenario(DataDrivenScenario.java:69)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:122)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:101)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:172)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:104)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:70)
]]>
    </error>
  </testcase> <!-- user is able to enter all the data on find a flight page successfully -->
  <testcase name="user is able to enter all the data on find a flight page successfully" classname="com.qmetry.qaf.automation.step.client.DataDrivenScenario">
    <skipped/>
  </testcase> <!-- user is able to enter all the data on find a flight page successfully -->
</testsuite> <!-- com.qmetry.qaf.automation.step.client.DataDrivenScenario -->
